### ISSUE TITLE
Japanese stemmer returning undefined instead of token being parsed

### ISSUE THREAD

**Issue by niftylettuce** [NaturalNode/natural#584]
Ref: <https://github.com/NaturalNode/natural/blob/master/lib/natural/stemmers/stemmer_ja.js#L104>

It also I think is inconsistent with other stemmers in that it doesn't export a `.stem` function.

```sh
  PorterStemmerIt: {
    stem: [Function (anonymous)],
    tokenizeAndStem: [Function (anonymous)]
  },
  PorterStemmerNo: {
    stem: [Function (anonymous)],
    addStopWord: [Function (anonymous)],
    addStopWords: [Function (anonymous)],
    tokenizeAndStem: [Function (anonymous)],
    getR1: [Function: getR1],
    step1: [Function: step1],
    step1a: [Function: step1a],
    step1b: [Function: step1b],
    step1c: [Function: step1c],
    step2: [Function: step2],
    step3: [Function: step3]
  },
  PorterStemmerSv: {
    stem: [Function: stem],
    addStopWord: [Function (anonymous)],
    addStopWords: [Function (anonymous)],
    tokenizeAndStem: [Function (anonymous)]
  },
  PorterStemmerPt: {
    stem: [Function (anonymous)],
    addStopWords: [Function (anonymous)],
    tokenizeAndStem: [Function (anonymous)]
  },
  PorterStemmerNl: PorterStemmer {},
  LancasterStemmer: {
    stem: [Function (anonymous)],
    addStopWord: [Function (anonymous)],
    addStopWords: [Function (anonymous)],
    removeStopWord: [Function (anonymous)],
    removeStopWords: [Function (anonymous)],
    tokenizeAndStem: [Function (anonymous)]
  },
  StemmerJa: [Function: StemmerJa],
```

```sh
> require('natural').StemmerId.stem('foo')
'foo'
> require('natural').StemmerJa('foo')
undefined
```

**Comment by Hugo-ter-Doest** [comment]
I think it is because the Japanese stemmer exports the constructor function to which you have to say `new`. The other stemmers export with `new stemmer()`. I'm going to repair this and make `stemmer_ja` export the actual stemmer object. Then it is consistent with the other stemmers. This will be a major release of course.

**Comment by Hugo-ter-Doest** [comment]
Fixed in #585

**Comment by niftylettuce** [comment]
many thanks @Hugo-ter-Doest ❤️
